static/templates/admin_playground_list.hbs

```diff
@@ -9,5 +9,12 @@
     <td>
         <span class="playground_url_prefix">{{url_prefix}}</span>
     </td>
+    {{#if ../can_modify}}
+    <td class="no-select actions">
+        <button class="button small delete btn-danger" data-playground-id="{{id}}">
+            <i class="fa fa-trash-o" aria-hidden="true"></i>
```

Comment:
This should be `aria-label` + `title` -- you want `aria-hidden` for icons that are decorations to a textual label for a button; and aria-label+title when the icon is the entire button.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA has background for this.